src/Bitcoin/FeeCalculator.cpp

```diff
@@ -34,9 +34,10 @@ class DecredFeeCalculator : public LinearFeeCalculator {
         : LinearFeeCalculator(gDecredBytesPerInput, gDecredBytesPerOutput, gDecredBytesBase) {}
 };
 
-constexpr DefaultFeeCalculator defaultFeeCalculator{};
-constexpr DecredFeeCalculator decredFeeCalculator{};
-constexpr SegwitFeeCalculator segwitFeeCalculator{};
+// note: c++20 seems not supported yet for constexpr destructor for swift build
```

Comment:
we have iOS build with c++20, can you double check?

Reply:
I will check thanks

Reply:
Fixed in [716e74e](https://github.com/trustwallet/wallet-core/pull/2397/commits/716e74e5e65e4331926d39ecbea32182def87a48)